models/treasures.models.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection")
+
+exports.selectTopics = () => {
+return db.query("SELECT * FROM topics")

```

Comment:
This is a good start, but what do we get back from this promise? We ideally want to send back the final version of the data to the controller.